src/auth/dto/changeInfo-dto.ts

```diff
@@ -0,0 +1,26 @@
+import { IsEmail, IsNotEmpty, IsString, MaxLength, MinLength, IsNumber } from 'class-validator';
+
```

Comment:
use a more descriptive name: pvz "change-user-info-dto.ts" and the actual class to "ChangeUserInfoDto", I know it may seem like a small change but changeInfo, doesn't really describe anything, because it can be any info for any object